frontends/web/src/api/account.ts

```diff
@@ -239,3 +239,7 @@ export interface IFeeTargetList {
 export const getFeeTargetList = (code: AccountCode): Promise<IFeeTargetList> => {
     return apiGet(`account/${code}/fee-targets`);
 };
+
+export const verifyAddress = (code: AccountCode, addressID: string): Promise<null> => {
```

Comment:
Doesn't it return `Promise<boolean>` ?

Reply:
True, fixed. I didn't even check because the result is unused.